### PR TITLE
weka: 3.6.12 -> 3.8.0

### PR DESCRIPTION
**Pull request by mimame**
This is my first pull request to NixOS so I hope that I have done it well.
###### Motivation for this change

Update weka to the last stable version.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/math/weka/default.nix [hunk 1]
```diff
@@ -1,29 +1,30 @@
 { stdenv, fetchurl, jre, unzip }:
 
-stdenv.mkDerivation {
-  name = "weka-3.6.12";
-  
+stdenv.mkDerivation rec {
+  name = "weka-${version}";
+  version = "3-8-0";
```
Comment (joachifm) [inline pkgs/applications/science/math/weka/default.nix]: Please use `3.8.0`; to reduce duplication, you can do `replaceChars ["."] ["-"]`.

**Comment by joachifm** [comment]
Other than my note, this looks superficially okay to me.

**Comment by mimadrid** [comment]
Thanks @joachifm for the advise. I think is ready now.

**Comment by mimadrid** [comment]
@joachifm, I've noticed that this package hasn't a maintainer. Could be I one? I'm very new to NixOS but I think one newbie is better than no one.

**Comment by joachifm** [comment]
@mimadrid that'd be great. All that is required to be a maintainer is caring about the package.

**Comment by joachifm** [comment]
Pushed to master. Thank you.

**Pull request closed without merging**